apps/judicial-system/web/src/routes/Prosecutor/Indictments/Defendant/PoliceCaseInfo.tsx

```diff
@@ -62,6 +63,7 @@ export const PoliceCaseInfo: React.FC<Props> = (props) => {
     setPoliceCase,
     deletePoliceCase,
     updatePoliceCases,
+    policeCaseNumberImmutable = false,
```

Comment:
```suggestion
    isPoliceCaseNumberImmutable = false,
```
Consider adding prefix to boolean variable